_posts/2017-09-26-plugin-api-alpha.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Pulp 3 Plugin API Alpha
+author: Michael Hrivnak
+tags:
+    -3.0
+---
+
+You are invited to try the first alpha release of the Pulp 3 Plugin API! Pulp 3
```

Comment:
No links to PyPI in this post. How about to add it for `the Pulp 3 Plugin API` ?
https://pypi.python.org/pypi/pulpcore-plugin